source/index-management.rst

```diff
@@ -682,6 +682,11 @@ Common API Components
      * server-side is used.
      */
     collation: Document;
+
+    /**
+     * Optionally specifies the wildcard projection of a wildcard index.
```

Comment:
This may be outside the scope of the PR, but I think it's be beneficial to remove "Optionally" from the prefix of each doc block description and instead wrap the types with `Optional<TYPE_NAME>` as we do in other specifications. Perhaps we can do this after merging this PR.